packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -156,7 +156,9 @@ class DefaultTextEditingShortcuts extends Shortcuts {
   }) : super(
     key: key,
     debugLabel: '<Default Text Editing Shortcuts>',
-    shortcuts: _shortcuts,
+    shortcuts: kIsWeb
+        ? <LogicalKeySet, Intent>{ ..._shortcuts, ..._webAdditionalShortcuts }
+        : _shortcuts,
```

Comment:
Not sure what the right balance is between consistency and feature completeness, but it looks like our `_macShortcuts` is not complete. macOS supports a subset of Emacs key bindings, e.g. Ctrl + A (go to start of line), Ctrl + E (go to end of line), Ctrl + F (one char forward), Ctrl + B (one char backward), etc. I don't see them in our list.

Reply:
I'm not sure if this is the right place to ask this. But I have a Flutter Web app deployed on Firebase,  
When on an android, chrome and safari, everything is fine.   But on IOS,   The space button does nothing when trying to add a space input in a TextField.    ?